### PR TITLE
Setting urllib version and refactoring robustscaler tests

### PR DESCRIPTION
**Pull request by lastmansleeping**

### CODE REVIEW

File: python/ml4ir/base/tests/test_robust_scaler_transform.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def test_reciprocal_rank_negative_scores(self):
 
         self.assertTrue(np.isclose(actual_reciprocal_ranks, expected_reciprocal_ranks).all())
 
-    def test_reciprocal_rank_zeros(self):
+    def test_robust_scaler_zeros(self):
```
Comment (marjanhs) [inline python/ml4ir/base/tests/test_robust_scaler_transform.py:36]: what is the reason behind renaming ?

File: python/requirements.txt [hunk 2]
```diff
@@ -4,6 +4,7 @@ numpy==1.24.2
 pandas==1.5.3
 scipy==1.10.0
 PyYAML==5.4.1
+urllib3<2.0  # Issue -> https://stackoverflow.com/questions/76187256/importerror-urllib3-v2-0-only-supports-openssl-1-1-1-currently-the-ssl-modu
```
Comment (marjanhs) [inline python/requirements.txt:7]: do you now why did not have it before ?

Comment (lastmansleeping) [reply python/requirements.txt:7]: I think the issue is coming from internal PyPi. Someone probably uploaded urllib3>2.0 recently and now it is auto installing that latest version.

Comment (lastmansleeping) [reply python/requirements.txt:7]: No, it was installing 2.1.0 earlier too. I am confused now as well.

File: python/ml4ir/base/tests/test_robust_scaler_transform.py (hunk 1 above)
Comment (lastmansleeping) [reply python/ml4ir/base/tests/test_robust_scaler_transform.py:36]: I think the naming was just wrong earlier. Probably a copy-paste error. Can you confirm @mohazahran ?

**Review by marjanhs — APPROVED** [review]

**Pull request merged**